etna/ciim/tests/factories.py

```diff
@@ -3,65 +3,77 @@
 
 def create_record(
     iaid="C0000000",
-    source="mongo",
+    admin_source="mongo",
     reference_number="ADM 223/3",
-    title="Title",
+    summary_title="Summary Title",
     description="description",
     earliest="1900",
     latest="2100",
     is_digitised=False,
     media_reference_id="0f183772-6fa7-4fb4-b608-412cf6fa8204",
     hierarchy=None,
     related=None,
+    hits_source_key_value_list=None,
```

Comment:
How do you feel about `source_value_overrides` as a name, and adding a type hint here to indicate that an `Optional[Dict[str, Any]]` is expected?